co-op/modal.html

```diff
@@ -22,9 +22,11 @@
         </div>
         <div class="description" style="width:750px">
             <div class="ui header">{{ member.summary }}</div>
-            <p>{{ member.statement }}</p>
+            <p>{{ member.term }}</p>
+            <p>{{member.skills}}</p>

```

Comment:
Do you think it would be better to move the "Tools:" portion here? so the `skills` entry in the `member` object would look like:
```
- skills: TypeScript, React, Ruby on Rails
```

And then in `modal.html` we could have 
```
<p>Tools: {{member.skills}}</p>
```

We could also refactor it to be `tools` or technologies rather than `skills`

Reply:
changes madee